pages/reg_page.php

```diff
@@ -1,84 +1,113 @@
+<?php 
+    $mysql = new mysqli('localhost','root','','hl_blog');
+    $content = $mysql->query("SELECT * FROM users");
+?>
+
 <!DOCTYPE html>
 
 <html>
 
 <head>
-    <meta charset="utf-8">
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Abel&display=swap" rel="stylesheet">
+    <?php require "../sections/head.php" ?>
     <link rel="stylesheet" href="/css/style_reg_page.css">
     <link rel="stylesheet" href="/css/header-footer.css">
-    <link rel="icon" href="/img/logo.png" type="image/png">
+    <link rel="stylesheet" href="/css/same_styles_log-reg.css">
     <title>Registration</title>
-
 </head>
 
 <body>
 
     <div>
-        <header>
-            <div>
-                <a class="logotype" href="/"><img src="/img/logo.png" class="img-logo"></a>
+        <?php require "../sections/header.php" ?>
+        <?php
+            if (isset($_POST['username'])) {
+                $username = stripslashes($_REQUEST['username']);
```

Comment:
i think here you can do this in 1 line, instead of 2 lines.  Concatenate this 2 lines in 1, by nesting functions in order